agent.py

```diff
@@ -0,0 +1,63 @@
+import typer
+import time
+import logging
+from decimal import Decimal
+
+import prediction_market_agent as pma
+from prediction_market_agent.markets.all_markets import MarketType, get_binary_markets
+from prediction_market_agent.agents.abstract import AbstractAgent
+from prediction_market_agent.agents.all_agents import AgentType, get_agent
+from prediction_market_agent.tools.utils import should_not_happen, check_not_none
+from prediction_market_agent.tools.types import Mana, xDai
+
+
+def main(
```

Comment:
There is a new agent.py entry point for infinite runs, and a bunch of TODOs are still here for following PRs .